department-of-veterans-affairs/caseflow
# APPEALS-42525 - Package Details: General Information

Resolves [APPEALS-42525 - Package Details: General Information](https://jira.devops.va.gov/browse/APPEALS-42525)

# Description
This adds the General Information under the Package Details tab of the Correspondence Details page.

## Acceptance Criteria

## Testing Plan

1. Navigate to Correspondence Cases page
2. Click on Unassigned tab
3. Select any Correspondence, which will take you to the Review Package page(must not have a pending request [banner])
4. Select a Correspondence Type on the Review package page under the Notes section on the right and Save changes.
5. In url bar, remove the "review_package" from the url to navigate to that correspondence's detail page.
6. Click on "Package Details" tab
7. Validate the information is populating appropriately
    1. Veteran Details
    2. Correspondence Type (should match what you selected on Review Package page)
    3. Package Document Type (NOD or Non-NOD)
    4. VA Date of Receipt
    5. Notes
8. Repeat with different Correspondences and Correspondence Types, as necessary.

**Note**: Future stories will limit access to details page to include only Pending/Completed, which should all have Correspondence Types.  Some seed data may have blank types.

# Frontend
## User Facing Changes

### Correspondence Details
![Screenshot 2024-07-31 at 4 41 23 PM](https://github.com/user-attachments/assets/8b31d634-cd6e-4c27-8ebb-a246b48cf7db)

### CSS/Wireframe Testing
![Screenshot 2024-07-31 at 4 45 42 PM](https://github.com/user-attachments/assets/fc3034e3-c2f1-4460-a22d-f091b8ba20a9)
![Screenshot 2024-07-31 at 4 45 59 PM](https://github.com/user-attachments/assets/d8e790e7-a51c-4f34-861a-c3d164f3ef9c)
![Screenshot 2024-07-31 at 4 46 12 PM](https://github.com/user-attachments/assets/e2b5b319-d3f3-4661-b34d-0e6ec598e97c)

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
